GirslShopping/src/test/java/com/skillrary/girslshopping/testscripts/TC005.java

```diff
@@ -0,0 +1,10 @@
+package com.skillrary.girslshopping.testscripts;
+
+public class TC005 {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```

Comment:
Why is main method empty??

Reply:
Sorry Hemanth Sir, I fixed it now, Please check now